kubernetes/cluster_loadtest.go

```diff
@@ -57,6 +57,10 @@ func (c *Cluster) bulkLoad(loadtestPod string, appPod string) error {
 		return err
 	}
 
+	// If this command fails, assume user was already created
+	cmd = exec.Command("kubectl", "exec", appPod, "--", "./bin/platform", "user", "create", "--email", "success+ltadmin@simulator.amazonses.com", "--username", "ltadmin", "--password", "ltpassword", "--system_admin")
```

Comment:
If it does happen to fail, the reasons might be mysterious if we don't log something, akin to the `xmlsec1` issues recently resolved. Can we dump `cmd.CombinedOutput()` somewhere?

Reply:
Yep, I can log it out